libnopegl/src/backends/gl/gpu_ctx_gl.c

```diff
@@ -148,7 +148,7 @@ static void reset_capture_cvpixelbuffer(struct gpu_ctx *s)
 }
 #endif
 
-static int create_texture(struct gpu_ctx *s, int format, int32_t samples, struct texture **texturep)
+static int create_texture(struct gpu_ctx *s, int format, int32_t samples, int usage, struct texture **texturep)
```

Comment:
`practise`/`practice` typo in commit description

Reply:
Fixed.